boxoffice/models/discount_policy.py

```diff
@@ -137,6 +193,11 @@ def __init__(self, *args, **kwargs):
     discount_policy_id = db.Column(None, db.ForeignKey('discount_policy.id'), nullable=False)
     discount_policy = db.relationship(DiscountPolicy, backref=db.backref('discount_coupons', cascade='all, delete-orphan'))
 
+    def update_used_count(self):
+        from ..models import LineItem, LINE_ITEM_STATUS
```

Comment:
This import can be at the top of the file. It should work as expected. It will also work if written as `from . import LineItem, LINE_ITEM_STATUS`.